src/GovUk.Education.ExploreEducationStatistics.Admin/ViewModels/UserInviteRequestViewModel.cs

```diff
@@ -0,0 +1,12 @@
+using System.ComponentModel.DataAnnotations;
+
+#nullable enable
+namespace GovUk.Education.ExploreEducationStatistics.Admin.ViewModels
+{
+    public class UserInviteRequestViewModel
```

Comment:
Same as above - would call this either `UserInviteViewModel` or `UserInviteRequest`

Reply:
I think this change has been in response to my comment that we now seem to have a convention of suffixing all the request models `ViewModel`. I'd been thinking `PreReleaseUserInviteViewModel` and `UserInviteViewModel` would be best.

Reply:
After Slack discussing, renaming to follow `BlahViewModel` convention